internal/driver/glfw/window_desktop.go

```diff
@@ -314,7 +314,10 @@ func (w *window) detectScale() float32 {
 		return 1.0
 	}
 
-	widthMm, _ := monitor.GetPhysicalSize()
+	widthMm, heightMm := monitor.GetPhysicalSize()
+	if widthMm == 60 && heightMm == 60 { // some sort of failure - mostly on Steam Deck
```

Comment:
Where does these `60` come from? AFAIU the GLFW source, it would return `0` if the monitor did not provide a number.
If these values are based on experience, the comment should explain this.

Reply:
Also, this might result in a wrong scale for real-life 60mmx60mm displays with a huge DPI 😁.

Reply:
That is true. If there was ever a 6cm square screen that we had to support. If that ever happens I will update this for sure!